internal/host/validator.go

```diff
@@ -1626,3 +1634,38 @@ func (v *validator) noSkipMissingDisk(c *validationContext) (ValidationStatus, s
 	}
 	return ValidationSuccess, successMessage
 }
+
+
+
+func (v *validator) serviceCanConnectToSpokeKubeAPI(c *validationContext) (ValidationStatus, string) {
+	// When assisted-service operates with kubeApiEnabled for a day2 cluster, it will automatically sign CSR's by making a connection from the assisted-service to the spoke kubeApi. 
```

Comment:
```suggestion
	// When assisted-service operates in kube-api mode, when adding a day2 host, when CSRs are not otherwise automatically approved, the service will automatically attempt to approve CSRs by using its stored spoke kubeconfig 
```

Reply:
Which also means we only need this validation when the spoke is UMN

Reply:
However even in non-UMN we (shoulud) track the installation progress using List nodes, so at-least those permissions are always needed regardless of UMN

Reply:
Perhaps this entire text should be simplified to something a bit less specific like "When adding a day2 host to a spoke cluster and the service operates in kube-API mode, the service needs to access the spoke cluster's kube API in order to perform some operations, such as listing nodes and approving CSRs (depending on platform). This validation makes sure in advance that the service is authorized to do that"